### ISSUE TITLE
Annual values in Well and Monitoring data

### ISSUE THREAD

**Issue by EliVolsch** [kartoza/IGRAC-GGIS#696]
Priority: Medium
Good morning,

I noticed that when downloading data from Czech Republic from Well and Monitoring data, the files contains annual averages of groundwater levels, while if I download the same dataset from the GGMN, I get weekly values (as it should be since the data is originally like this). Why is this happening? The data should not be aggregated but displayed as it was originaly uploaded. I am also wondering if this happening in other countries as well.

**Comment by meomancer** [comment]
Hi @clau1313 cc @Eliegergesgerges 
This is because we have 2 organisations under Czech country
1. Czech Republic - Czech Hydrometeorological Institute
2. Jasechko et al. (2024) Rapid groundwater decline and some cases of recovery in aquifers globally

You can see on the "Well and Monitoring data", it has those organisations.
![Selection_029](https://github.com/kartoza/IGRAC-GGIS/assets/4530905/7b27509c-1f43-4d83-9a5c-3fa71c2474d7)

Everything with name "7000" are from Jasechko, which has annual data
![Selection_032](https://github.com/kartoza/IGRAC-GGIS/assets/4530905/fb6c4948-b625-4356-8aac-837771770ae8)

Meanwhile with name "VB" are from "Czech Republic - Czech Hydrometeorological Institute" which has weekly data
![Selection_033](https://github.com/kartoza/IGRAC-GGIS/assets/4530905/d0c5502f-5a56-4ad8-8b39-a30fe72619e4)


So basically, when we download "Well and Monitoring Data", it returns both of organisations.

Meanwhile, on GGMN, it just returns "Czech Republic - Czech Hydrometeorological Institute", because below configurations
![Selection_034](https://github.com/kartoza/IGRAC-GGIS/assets/4530905/3a3e1c9e-074b-41f8-b8da-b54e31433fb3)

That why on GGMN, it just return weekly data

**Comment by clau1313** [comment]
Thanks @meomancer! All clear now.
